.Rbuildignore

```diff
@@ -23,3 +23,6 @@
 ^data-raw$
 ^LICENSE\.md$
 ^\.github$
+^metalite\.Rcheck$
```

Comment:
Instead of bloating up `.Rbuildignore` further, I'm wondering how these assets are created.

Using `R CMD build` and `R CMD check`, they are commonly created at the same level of the source directory.

Reply:
It was created automatically when I open the project in Rstudio Cloud.

Reply:
Good to know - maybe there is a good reason for them, but they really need to stop doing random things like this.

I created a PR to remove all the unused patterns in both files: https://github.com/Merck/metalite/pull/40